src/pages/404.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
```

Comment:
Maybe we can work with Hillary to get a similar feel we have on the main site.

https://theartofeducation.edu/nothing-here